### ISSUE TITLE
PyPi broken for cloudformation-cli-go-plugin?

### ISSUE THREAD

**Issue by x6j8x** [aws-cloudformation/cloudformation-cli-go-plugin#165]
PyPi doesn't seem include the newest versions (1.0.0, 1.0.1) of the CloudFormation-cli-go-plugin

```
$ pip3 install cloudformation-cli-go-plugin==
ERROR: Could not find a version that satisfies the requirement cloudformation-cli-go-plugin== (from versions: 0.1, 0.1.1, 0.1.2, 0.1.3, 0.1.4, 0.1.5, 0.1.6, 2.0.0)
```

**Comment by brianterry** [comment]
Thanks, @x6j8x for submitting this issue. We will be pushing a release to Pypi shortly.

**Comment by omkhegde** [comment]
Hi @x6j8x , we have just released package version 2.0.1 on PyPi (https://pypi.org/project/cloudformation-cli-go-plugin/2.0.1/). Please note that 1.0.0 and 1.0.1 in your question above are not valid package versions for the plugin. The plugin has version 2.0.0 and 2.0.1.
